### ISSUE TITLE
Personal Physics v5: formal expressivity ladder and MiniJev typed decision engine

### ISSUE THREAD

**Issue by jwalin-shah** [jwalin-shah/tensor-logic#89]
## Goal

Separate three claims rigorously:

1. what the current tensor-logic engine can express and prove today,
2. what additional primitives would be required for computational universality,
3. whether a Jev-like typed probabilistic decision surface can be implemented using Tensor Logic equations and calibrated training.

## Expressivity ladder

Prove with executable tests / references:

L0 finite tensor algebra
- scalar, sum, product, projection/einsum, difference, step, sigmoid

L1 relational algebra / conjunctive queries
- joins, existential projection, union

L2 positive Datalog on finite domains
- recursive fixed-point / transitive closure
- monotone convergence

L3 weighted / real-valued programs
- weighted sums
- logistic decisions
- differentiable tensor equations

L4 neural primitives
- affine layer
- MLP
- attention-style contraction
- shared representation + parallel heads

L5 typed System-One decision surface
- Choice(options)
- Score(ordered levels)
- Noul(probability yes/no)
- batch multiple questions in parallel
- calibrated probabilities

L6 universality research
- DO NOT claim current engine Turing complete
- formalize which missing mechanism is required: unbounded/dynamically extensible memory, unrestricted recursion/control, or a published reduction from a universal computation model
- reproduce a small universal machine only if semantics are explicit and the proof does not depend on hidden Python control flow

## MiniJev v0

Build a model interface:

state tensor -> shared representation H

Choice:
logits[q,o] = einsum(H, W_choice[q,o,:]) + b
p = softmax(logits)

Score:
p(level | state); score = expectation over ordered levels

Noul:
p_yes = sigmoid(dot(H, w_q) + b_q)

Requirements:
- outputs always typed
- confidence/probability distribution always exposed
- multiple questions evaluated in one batch
- no free-form generation
- calibration metrics: NLL, Brier, ECE
- synthetic benchmark with train/validation/test split
- compare direct tensor decision head vs hand rules vs a simple MLP baseline
- record latency for 1, 10, 100 parallel questions
- preserve provenance from state features to outputs where feasible

## Important claim boundary

Passing this issue proves Tensor Logic can host a Jev-like decision primitive/interface. It does NOT prove equivalence to TypeSafe Jev's undisclosed architecture, training data, RLCD method, or frontier task performance.

## Acceptance

- [ ] executable expressivity matrix
- [ ] transitive closure/fixpoint proof
- [ ] affine/MLP/parallel-head tests
- [ ] Choice/Score/Noul API
- [ ] synthetic calibration benchmark
- [ ] latency benchmark
- [ ] explicit non-Turing-completeness statement for current finite-domain engine
- [ ] universality proof only if a formal reduction is actually implemented and documented
